src/os/time/time.js

```diff
@@ -25,6 +25,9 @@ os.time.Duration = {
   WEEK: 'week',
   MONTH: 'month',
   YEAR: 'year',
+  LAST7DAYS: 'last 7 days',
```

Comment:
I believe we want last 24/48 hours as options. With dense vector data these would be the most common selections.